### PR TITLE
#2301 - Input new 2023/2024 Program Year Amounts - PT

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Updated values as per ACs;
- No test needed to change.

### CODE REVIEW

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn [hunk 1]
```diff
@@ -940,34 +940,34 @@
           <text>6</text>
         </inputEntry>
         <outputEntry id="LiteralExpression_1h90dwg">
-          <text>78634</text>
+          <text>86784</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_0lfggsr">
-          <text>0.062400</text>
+          <text>0.043680</text>
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn:946]: as per the screenshot in the story, there is no `0` in the end, 
image

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn [hunk 2]
```diff
@@ -978,34 +978,34 @@
...
-          <text>144457</text>
+          <text>153261</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_06qu2lc">
-          <text>84933</text>
+          <text>93737</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_04uliw4">
-          <text>0.00134400</text>
+          <text>0.000940800</text>
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn:993]: same here, 
image

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn [hunk 3]
```diff
@@ -902,34 +902,34 @@
...
-          <text>71782</text>
+          <text>79222</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_0rv7ikh">
-          <text>0.00145066</text>
+          <text>0.001015462</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1fhqv4y">
-          <text>0.00217600</text>
+          <text>0.001523200</text>
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn:920]: https://github.com/bcgov/SIMS/pull/2611/files#r1434445784

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn [hunk 4]
```diff
@@ -864,34 +864,34 @@
...
-          <text>64204</text>
+          <text>70859</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1ss5azf">
-          <text>0.00151466</text>
+          <text>0.001060262</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1t2eqs6">
-          <text>0.00227200</text>
+          <text>0.001590400</text>
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn:882]: https://github.com/bcgov/SIMS/pull/2611/files#r1434449993

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn [hunk 5]
```diff
@@ -978,34 +978,34 @@
...
-          <text>84933</text>
+          <text>93737</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_04uliw4">
-          <text>0.00134400</text>
+          <text>0.000940800</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_18gg2hi">
-          <text>0.00201600</text>
+          <text>0.001411200</text>
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn:996]: https://github.com/bcgov/SIMS/pull/2611/files#r1434448511

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn [hunk 6]
```diff
@@ -788,34 +788,34 @@
...
-          <text>89004</text>
+          <text>93709</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1twitak">
-          <text>45399</text>
+          <text>50104</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_10l09c2">
-          <text>0.04128</text>
+          <text>0.041279670</text>
```
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn:815]: https://github.com/bcgov/SIMS/pull/2611/files#r1434450303

**Review by ann-aot — COMMENTED** [review]
LGTM @andrepestana-aot 👍  , just a qn added regarding the trailing 0s

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn (hunk 1 above)
Comment (andrepestana-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn]: That doesn't make any difference in the calculations but I removed them.

**Review by ann-aot — APPROVED** [review]
Thanks @andrepestana-aot for doing the changes. Nice work 👍

**Review by lewischen-aot — APPROVED** [review]
Thanks for making the changes. Nice work 👍

**Review by guru-aot — APPROVED** [review]
LGTM,nice work @andrepestana-aot

**Pull request merged**